worlds/witness/player_logic.py

```diff
@@ -401,7 +443,11 @@ def make_options_adjustments(self, world: "WitnessWorld") -> None:
 
         # Exclude panels from the post-game if shuffle_postgame is false.
         if not world.options.shuffle_postgame:
```

Comment:
Does this matter if the goal is panel_hunt / should this be `if not (world.options.shuffle_postgame or victory == "panel_hunt")`

Reply:
True